treeherder/webapp/api/runnable_jobs.py

```diff
@@ -16,6 +20,19 @@ def list(self, request, project):
         """
         GET method implementation for list of all runnable buildbot jobs
         """
+        taskID = request.query_params['taskID']
+        tc_jobs_url = "https://public-artifacts.taskcluster.net/" + taskID + "/public/full-task-graph.json"
+        tc_graph = None
+        validate = URLValidator()
+        try:
+            validate(tc_jobs_url)
+            resp = requests.get(url=tc_jobs_url)
```

Comment:
we have a function called ``etl/common/fetch_json()`` that would be good to use here.  It sets our useragent and handles a few things uniformly.  It would be better to use that here